frontend/src/store.js

```diff
@@ -74,7 +75,7 @@ const StateProvider = ({ children }) => {
       case "reset-current-question-info":
         return {
           ...state,
-          currentQuestionInfo: JSON.parse(JSON.stringify(baseQuestionInfo)),
+          currentQuestionInfo: Object.assign({}, baseQuestionInfo),
```

Comment:
Did you copy an old branch ?

Reply:
It was weird because of the force push. I fixed it above but missed this one